Vagrantfile

```diff
@@ -3,5 +3,16 @@ Vagrant.configure("2") do |config|
   config.vm.synced_folder ".", "/app"
   config.vm.provision "shell", inline: "/app/bin/install-ansible.sh", upload_path: "/home/vagrant/install-ansible.sh"
   config.vm.provision "shell", inline: "cd /app/ansible && ansible-playbook  -l localhost bakery.yml app-AfterInstall.yml app-StartServer.yml", upload_path: "/home/vagrant/apl.sh"
+  config.vm.provision "shell", inline: "curl -sSL https://rvm.io/mpapis.asc | gpg2 --import -"
```

Comment:
Ugh, don't do it like this, create a proper `bin/install-gauntlt.sh` script instead.

Reply:
Nevermind, I already did that in the PR I just merged in here. I forgot I even worked on this!

Reply:
Yeah, this was done a while back. Is this now a dead PR?

Reply:
This PR, #33 is still live and needs to get into master. It LGTM now, were you ready with this work and where it was or is there more to add before we merge it? 

I haven't tested this integration at all yet either.

Reply:
Consider where we would want to put these tests if we are running in CI, not just in Vagrant, too  @rpigu-i